### PR TITLE
[UTOPIA-983]update disabled btn can not hoverable

### PR DESCRIPTION
**Pull request by binggaofw**
# Description

This PR includes the following proposed change(s):

- see title
## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Development Dependency Working Agreement
- [x] My code DOES NOT include the importing of new dependencies into the DPIA ecosystem
- [ ] My code DOES include the importing of new dependencies into the DPIA ecosystem
**If new dependencies are being introduced to the DPIA ecosystem:**
- [ ] The functionality of the dependency drastically reduces code complexity and makes my changes more easily maintainable and readible 
- [ ] The dependency being introduced does not contain multiple layers of nested dependencies introducing maintainability complexity to the DPIA ecosystem

## Frontend Development Changes
- [ ] N/A
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to project documentation or diagrams that reflect my changes
- [ ] New and existing unit tests pass locally with my changes
- [ ] My code follows Airbnb React Style Guidelines

## API Development Changes
- [x] N/A
- [ ] I have performed a self-review of my own code
- [ ] My code follows standards and practices outlined in the BC Government API Development Guidelines
- [ ] New and existing unit tests pass locally with my changes
- [ ] My changes includes Swagger documentation updates that reflect the changes I am introducing

## Definition of Done

Definition of Done

### CODE REVIEW

File: src/frontend/src/sass/_buttons.scss [hunk 1]
```diff
@@ -103,6 +103,7 @@ a:hover {
 
   border: 2px solid $bg-disabled;
   cursor: auto;
+  pointer-events: none;
```
Comment (binggaofw) [inline src/frontend/src/sass/_buttons.scss:106]: this is the code I add, the rest is lint fix

Comment (kushal-arora-fw) [inline src/frontend/src/sass/_buttons.scss:106]: shouldn't we implement this instead?

```suggestion
  cursor: not-allowed;
```

Comment (kushal-arora-fw) [inline src/frontend/src/sass/_buttons.scss:106]: shouldn't we implement this instead?

```suggestion
  cursor: not-allowed;
```

File: src/frontend/src/sass/_form.scss [hunk 2]
```diff
@@ -107,7 +107,6 @@
   a {
     font-weight: normal;
     cursor: text;
-    user-select: text;
```
[the remark this answers is not in the document]
Comment (binggaofw) [reply src/frontend/src/sass/_form.scss:110]: yes, that's strange, I will add it back

**Review by kushal-arora-fw — DISMISSED** [review]

File: src/frontend/src/sass/_buttons.scss [hunk 3]
```diff
@@ -102,7 +102,7 @@ a:hover {
   --text-color: white;
 
   border: 2px solid $bg-disabled;
-  cursor: auto;
+  cursor: not-allowed;
```
Comment (kushal-arora-fw) [inline src/frontend/src/sass/_buttons.scss:105]:
```suggestion
  cursor: not-allowed;
  text-decoration: none;
```

**Review by kushal-arora-fw — APPROVED** [review]

**Pull request merged**